packages-automation/auto-cleanUpPayments/helpers/getAndpadPaymentsCsv.ts

```diff
@@ -0,0 +1,21 @@
+import { filePath } from '../config';
+import fs from 'fs';
+import Papa from 'papaparse';
+import { AndpadCsv } from '../types/types';
+
+
+/**
+ * ANDPADの入金一覧csvファイルを取得します
+ */
+export const getAndpadPaymentsCsv = () => {
+
+  const data = fs.readFileSync(filePath, 'utf8');
+  const result = Papa.parse(data, {
+    header: true,
+    dynamicTyping: true,
+    skipEmptyLines: true,
+  }) as AndpadCsv;
+  
+
+  return result;
+};
```

Comment:
```suggestion
export const getAndpadPaymentsCsv = () => {

  const data = fs.readFileSync(filePath, 'utf8');
  const result = Papa.parse<AndpadCsvData>(data, {
    header: true,
    dynamicTyping: true,
    skipEmptyLines: true,
  });
  
  return result;
};

export type GetAndpadPaymentsReturn = ReturnType<typeof getAndpadPaymentsCsv>;

```

- Papa.parseのGenericsに渡した型は、`result.data` の型になります。

- `GetAndpadPaymentsReturn ` は当関数の返り値の型です。これで、AndpadCsvを定義しなくても、利用出来ると思います。変わりに、paparse の `ParseResult<AndpadCsvData>` も利用出来ます。

Reply:
ご指摘ありがとうございます。
[a07f6c4](https://github.com/Lorenzras/yumecoco-monorepo/pull/889/commits/a07f6c4344f6497c0805ec22499e9426998d3427)にて修正しましたので、ご確認をお願いします。